llvm/include/llvm/SYCLLowerIR/DeviceConfigFile.td

```diff
@@ -33,7 +33,6 @@ def AspectExt_intel_gpu_slices : Aspect<"ext_intel_gpu_slices">;
 def AspectExt_intel_gpu_subslices_per_slice : Aspect<"ext_intel_gpu_subslices_per_slice">;
 def AspectExt_intel_gpu_eu_count_per_subslice : Aspect<"ext_intel_gpu_eu_count_per_subslice">;
 def AspectExt_intel_max_mem_bandwidth : Aspect<"ext_intel_max_mem_bandwidth">;
```

Comment:
Doesn't this aspect also apply to FPGA only?

Reply:
These are part of https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_intel_device_info.md extension which is not FPGA-specific, so I believe we need to save them.